src/renderer/entities/contact/model/contact.ts

```diff
@@ -1,8 +1,58 @@
-import { AccountId, Address } from '../../../domain/shared-kernel';
+import { createEffect, createEvent, createStore, forward } from 'effector';
 
-export type Contact = {
-  name: string;
-  address: Address;
-  accountId: AccountId;
-  matrixId?: string;
+import { ContactDS } from '@renderer/shared/api/storage';
+import { useContact } from '../lib';
+import type { Contact } from './types';
+
+export const $contacts = createStore<ContactDS[]>([]);
+const appStarted = createEvent();
+
+const populateContactsFx = createEffect(() => {
+  return useContact().getContacts();
```

Comment:
why do you decide to use `useContact()` and not extract `getContacts` function from it or create `contactStorage/Service` const?